code/modules/admin/topic.dm

```diff
@@ -55,33 +55,37 @@
 			return
 		switch(href_list["makeAntag"])
 			if("traitors")
-				if(src.makeTraitors())
-					message_admins("[key_name_admin(usr)] created traitors.")
-					log_admin("[key_name(usr)] created traitors.")
+				var/maxCount = input("Set number of Traitors","Set Traitor Count (max)",1) as num|null
```

Comment:
why set a default of 1? why not set it to 0 and have the make(whatever) procs handle it?

Reply:
Cancelling should correctly set null.
1 is the default since it is assumed you want at least 1, or to cancel.

Reply:
aight